agent/app/agent_capability_unix.go

```diff
@@ -236,3 +237,19 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+// var to allow mocking for checkNetworkTooling
+var isNetworkToolingAvailable = checkNetworkTooling
+
+// wrapper around exec.LookPath
+var lookPathFunc = exec.LookPath
+
+func checkNetworkTooling() bool {
```

Comment:
Let's please add a description for the function. Maybe we should rename it to make it clear that it looks for Fault Injection tooling and not network tooling in general.

Reply:
Implemented in https://github.com/aws/amazon-ecs-agent/commit/1a3ea5b7f2b55d50e8b41c84c10a75842c262893